cmd/server/login/otp/password.go

```diff
@@ -0,0 +1,36 @@
+package otp
+
+import (
+	"crypto/rand"
+	"math/big"
+)
+
+// Generator is an interface that generates a random string.
+type Generator interface {
+	Generate(length int) (string, error)
+}
+
+type GeneratorFunc func(length int) (string, error)
+
+func (g GeneratorFunc) Generate(length int) (string, error) {
+	return g(length)
+}
+
+// generateNumericOTP generates a numeric one-time password of the given length.
+func generateNumericOTP(length int) (string, error) {
+	const digits = "0123456789"
```

Comment:
Should we use only digits? Have more chars increases the entropy of the OTP, making it harder to bruteforce